apps/server/migrations/0020_deleted_record_insert.ts

```diff
@@ -22,11 +22,9 @@ const tables = [
   "tag",
   "test",
   "unit",
-  "unit_relation",
```

Comment:
why is this one removed

Reply:
i forgot to put it back after you fixed the deleted record stuff